canonical-web-and-design/react-components
# Prepare 0.37.2

## Done

- Prepare 0.37.2 release

## QA

- Make sure version is correct.
- Check if storybook renders fine
